docs/current/sailbot_workspace/docker_images.md

```diff
@@ -0,0 +1,23 @@
+# Sailbot Workspace Docker Images
+
+A table detailing the Docker images used to create the Dev Container can be found below.
+Click on an image to learn more about its features and how to update it.
+
+| Image                                         | Parent Image                                | Source Code                                                 | Why it is Rebuilt                     | Where it is Built |
+| --------------------------------------------- | ------------------------------------------- | ----------------------------------------------------------- | ------------------------------------- | ----------------- |
+| [`pre-base`][pre-base]{target=_blank}         | [Ubuntu 22.04][Ubuntu Image]{target=_blank} | [`pre-base.Dockerfile`][pre-base.Dockerfile]{target=_blank} | To update ROS or OMPL                 | Personal computer |
```

Comment:
Does the markdown link linter check links of this format? If so, it may conflict with footnotes

Reply:
No, it should not. If ever in doubt, you can take the regex expression:

https://github.com/UBCSailbot/docs/blob/6b5d0aeb8ab24e8ecd8be5bca4038a787efd6e0b/.github/actions/lint/lint_links.py#L8

and test it here: https://regex101.com/

If you get a match, that means that it will be checked. Make sure that the flavor is set to Python and not the default (PHP >= 7.3).

Reply:
We should make an issue for this, as it is another way to render links in Markdown

Reply:
Added to the tracker